stat/roc_example_test.go

```diff
@@ -125,3 +125,23 @@ func ExampleROC_aUC() {
 	// false positive rate: [0 0.5 0.5 1 1]
 	// auc: 0.25
 }
+
+func ExampleTOC() {
+	classes := []bool{
+		false, true, false, false, true, false,
+		true, false, false, true, true, true,
+		true, true, true, true, false, false,
+		false, false, false, false, false, false,
+		false, false, false, false, false, false,
+	}
+
+	min, toc, max := stat.TOC(classes)
+	fmt.Printf("minimum bound: %v\n", min)
+	fmt.Printf("TOC:           %v\n", toc)
+	fmt.Printf("maximum bound: %v\n", max)
+
+	// Output:
+	// minimum bound: [0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 2 3 4 5 6 7 8 9 10]
+	// TOC:           [0 0 1 1 1 2 2 3 3 3 4 5 6 7 8 9 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10]
+	// maximum bound: [0 1 2 3 4 5 6 7 8 9 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10]
+}
```

Comment:
I would add an example of using SortWeightedLabeled or SortLabeled as in the ROC examples.

Reply:
Done